databases/week3/SQL/queries.sql

```diff
@@ -0,0 +1,157 @@
+SET NAMES utf8mb4;
+USE meal_sharing_website;
+
+----------------------- MEAL TABLE --------------------
+
+-- Get all meals
+SELECT * FROM meal;
+
+-- Add a new meal
+INSERT INTO meal (title, description, location, `when`, max_reservations, price, created_date) 
+VALUES ('Spaghetti Bolognese', 'A classic Italian dish made with pasta and meat sauce', 'Ristorante Italiano', '2022-01-15 19:00:00', 10, 30.33, '2022-01-30');
+
+INSERT INTO meal (title, description, location, `when`, max_reservations, price, created_date) 
+VALUES ('Pad Thai', 'A popular Thai dish made with stir-fried noodles, vegetables, and peanuts', 'Thai Kitchen', '2022-03-20 18:30:00', 3, 18.99, '2022-02-28');
+
+INSERT INTO meal (title, description, location, `when`, max_reservations, price, created_date) 
+VALUES ('Vegan Burrito Bowl', 'A healthy and flavorful dish made with rice, beans, and veggies', 'The Green Bowl', '2022-04-11 12:00:00', 6, 12.50, '2022-04-15');
+
+INSERT INTO meal (title, description, location, `when`, max_reservations, price, created_date) 
+VALUES ('Steak and Lobster', '8 oz. filet mignon and 6 oz. lobster tail', 'The Steakhouse at the Ritz', '2022-10-01 19:30:00', 4, 89.99, '2022-10-28');
+
+INSERT INTO meal (title, description, location, `when`, max_reservations, price, created_date) 
+VALUES ('Sushi Platter', 'Assortment of fresh sushi rolls and sashimi', 'Sakura Japanese Restaurant', '2022-11-12 20:00:00', 6, 42.50, '2022-11-20');
+
+INSERT INTO meal (title, description, location, `when`, max_reservations, price, created_date) 
+VALUES ('Roasted Chicken and Vegetables', 'A hearty and healthy dish made with roasted chicken and seasonal vegetables', 'The Rustic Kitchen', '2023-01-10 18:30:00', 5, 24.99, '2023-01-18');
+
+INSERT INTO meal (title, description, location, `when`, max_reservations, price, created_date) 
+VALUES ('Shrimp Scampi', 'A delicious and easy-to-make Italian dish made with garlic, butter, and shrimp', 'Bella Vita', '2023-01-12 20:00:00', 8, 29.99, '2023-01-30');
+
+INSERT INTO meal (title, description, location, `when`, max_reservations, price, created_date) 
+VALUES ('Taco Tuesday', 'A fun and flavorful Mexican-inspired meal with tacos, chips, and guacamole', 'La Taqueria', '2023-02-22 17:00:00', 10, 19.99, '2023-02-25');
+
+-- Get a meal with any id, fx 1
+SELECT * FROM meal WHERE id = 1;
+
+-- Update a meal with any id
+UPDATE meal 
+SET title = 'Roasted Chicken', description = 'A classic dish made with juicy chicken roasted to perfection', location = 'The Roasting Company'
```

Comment:
It's more readable to put each value you’re updating on the separate line:)

Reply:
> Excellent🤩 you nailed it! Everything looks great and I can see your improvements.

Hello:) Thank you for your time and comments, i will modify the homework